src/js/main.js

```diff
@@ -2,13 +2,37 @@ jQuery.ajax('../apis/github/users/octocat.json').then(function(data){
   console.log(arguments);
 });
 
-// jQuery.ajax({})
+// http://www.jacklmoore.com/notes/jquery-tabs/
+// $('ul.tabs').on('click', function(){ // selects each element inside the ul tag
+//   var $active = $(this).find('a'); // `this` refers to containig element (ul.tabs); finds all `a` elements
+//   $active.addClass('active');
+// });
 
-// IFFE example
-// (function() {
-//   // the code here is executed once in its own scope
-// })();
+$('ul.tabs li a').on('click', function(e) {
```

Comment:
this is pretty complicated - can you think of a way to make this code DRY?

thanks for citing some resources!